src/ddmd/dsymbol.d

```diff
@@ -2007,6 +2029,78 @@ extern (C++) final class OverloadSet : Dsymbol
 }
 
 /***********************************************************
+ * Forwarding ScopeDsymbol.
+ * This is used by ForwardingAttribDeclaration and ForwardingScopeDeclaration
```

Comment:
s/This is used by/Used by/

(I'm a fan of Strunk & White https://www.amazon.com/Elements-Style-Classic-Writing-Guide/dp/1537669648).